src/Transport/obsoletes-v2.cs

```diff
@@ -0,0 +1,71 @@
+﻿#pragma warning disable 1591
```

Comment:
For the release branch, we don't need the obsoletes file. These should just be in `AzureServiceBusTransportSettingsExtensions`

Reply:
aye-aye, captain